controllers/kbsconfig_controller.go

```diff
@@ -359,6 +371,20 @@ func volumesToVolumeMounts(volumes []corev1.Volume) []corev1.VolumeMount {
 	return volumeMounts
 }
 
+// Method to add volumeMounts for KBS under custom directory
+func volumesToVolumeMountsCustom(volumes []corev1.Volume, mountPath string) []corev1.VolumeMount {
```

Comment:
Can we avoid having 2 functions doing the same thing? I think you can call `volumesToVolumeMountsCustom` with "/etc" mount path for the `kbsEtcVolumes`